etl-pipeline/processes/record_file_saver.py

```diff
@@ -61,10 +60,11 @@ def get_file_contents(self, file_url: str):
         except Exception as e:
             logger.exception(f'Failed to get file from {file_url}')
             raise e
-    
-    def generate_webpub(self, file_root: str):
-        file_path = get_stored_file_url(storage_name=self.file_bucket, file_path=f'{file_root}/META-INF/container.xml')
-        webpub_conversion_url = f'{RecordFileSaver.WEBPUB_CONVERSION_BASE_URL}/api/{quote_plus(file_path)}'
+
+    def generate_webpub(self, source_url: str):
+        webpub_conversion_url = (
+            f'{RecordFileSaver.WEBPUB_CONVERSION_BASE_URL}/api/{quote_plus(source_url)}'
+        )
```

Comment:
oh this makes no sense lol, I'll add some kind of fixture / mock webpub converter.